nautobot/core/testing/views.py

```diff
@@ -400,6 +403,22 @@ def test_create_object_with_constrained_permission(self):
                 else:
                     self.assertInstanceEqual(self._get_queryset().last(), self.form_data)
 
+        @override_settings(EXEMPT_VIEW_PERMISSIONS=["*"])
+        def test_extra_feature_form_fields_present(self):
+            model_class = self.model
+            model_form = lookup.get_form_for_model(model_class)
+            fields = model_form.base_fields
+            # Need to have actual custom field and relationship instances associated with all content-types
+            # in order for this generic test to check these fields.
+            # if isinstance(model_class, CustomFieldModel):
+            #     self.assertNotEqual(model_form.custom_fields, None)
+            # if isinstance(model_class, RelationshipModel):
+            #     self.assertNotEqual(model_form.custom_fields, None)
```

Comment:
Can we `assertTrue(issubclass(CustomFieldModelForm, model_form))` or something like that?

Reply:
`CustomFieldModelFormMixin` and `RelationshipModelFormMixin`?